<launch>
    
    <group ns="vins_1">
        <param name="config_file" type="string" value="$(find feature_tracker)/../config/euroc/realsense_435i.yaml" />
        <param name="vins_folder" type="string" value="$(find feature_tracker)/../config/../" />
        <param name="agent_num" type="int" value="1" />
        <node name="feature_tracker" pkg="feature_tracker" type="feature_tracker" output="log"/>
        <node name="vins_estimator" pkg="vins_estimator" type="vins_estimator" output="log"/>

        <include file="$(find realsense2_camera)/launch/my_camera.launch" />
        <include file="$(find rplidar_ros)/launch/rplidar.launch" />
        
        <!-- <node name="obs_detection" pkg="my_toolkit" type="obs_detection" output="screen"/>
        <node name="car_serial" pkg="my_toolkit" type="car_serial" output="screen">
            <param name="print_debug" type="bool" value="true"/>
        </node> -->

    </group>

</launch>